dags/mantaray/mantaray.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DAGs to run Mantaray benchmarks."""
+
+
+import datetime
+from airflow import models
+from xlml.utils import mantaray
+import yaml
+from dags import composer_env
+
+# Skip running this script in unit test because gcs loading will fail.
+if composer_env.is_prod_env() or composer_env.is_dev_env():
```

Comment:
nit, is it possible to print out in the else branch that we aren't running this since this is not in the right zone?

Reply:
sounds good!